specification/security/resource-manager/Microsoft.Security/preview/2023-06-01-preview/aggregations.json

```diff
@@ -0,0 +1,133 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Aggregations Query",
+    "description": "API spec for Microsoft.Security (Azure Security Center) Aggregations Query",
+    "version": "2023-06-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": ["https"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "security": [
+    {
+      "azure_auth": ["user_impersonation"]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Security/aggregations": {
+      "post": {
+        "tags": ["aggregations"],
+        "description": "Run pre-defined aggregation query against Microsoft.Security data types for scopes specified in the request.",
+        "operationId": "Aggregations",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "aggregations",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AggregationRequest"
+            },
+            "description": "Request specifying Aggregation type and scopes."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Result of the aggregation query operation",
+            "schema": {
+              "$ref": "#/definitions/AggregationResponse"
+            }
+          },
+          "default": {
+            "description": "An error occurred while processing the request. See the error.code parameter to identify the specific error.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Basic query": {
+            "$ref": "./examples/BasicResult_example.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AggregationRequest": {
+      "description": "Aggregation query type and scopes.",
+      "type": "object",
+      "properties": {
+        "azureScopes": {
+          "description": "The azure scopes in which the query will be executed - List of subscription IDs.",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "Scope."
+          }
+        },
+        "awsScopes": {
```

Comment:
doesn't it make sense the awsScopes and gcpScopes will be the accountIds and projectIds instead of the security connector ids?

Reply:
we prefer to keep the same definition as query api as we are doing exactly the same
we pass the connector id as scope to checkaccess api